src/player/Player.php

```diff
@@ -1374,6 +1378,14 @@ protected function processMostRecentMovements() : void{
 			$this->logger->debug("Exceeded movement rate limit, forcing to last accepted position");
 			$this->sendPosition($this->location, $this->location->getYaw(), $this->location->getPitch(), MovePlayerPacket::MODE_RESET);
 		}
+
+		if ($this->openContainerPosition !== null) {
```

Comment:
There should be an API for plugins to be able to disable this behavior on their custom inventories